scripts/build.release.sh

```diff
@@ -8,6 +8,6 @@ fi
 
 # git submodule add https://github.com/googleapis/googleapis
 # git submodule add https://github.com/ava-labs/avalanche-network-runner
-git submodule update --init --remote
```

Comment:
--remote flag pulls latest commit from the branch vs SHA pinned.

Gotta love submodules :upside_down_face: